recipes/rb-rack/meta.yaml

```
@@ -0,0 +1,40 @@
+{% set name = "rack" %}
+{% set version = "3.0.4" %}
+
+package:
+  name: rb-{{ name|lower }}
+  version: {{ version }}
+
+source:
+  - url: https://rubygems.org/downloads/{{ name }}-{{ version }}.gem
+    sha256: 93008e4e29af4ac220ec19ed814f288f0cc27e14b1a4b27216e6e4b7e1e73cf6
+  
+build:
+  number: 0
+  script:
+    - gem install -N -l -V --norc --ignore-dependencies {{ name }}-{{ version }}.gem
+    - gem unpack {{ name }}-{{ version }}.gem
+  skip: true  # [win]
+
+requirements:
+  host:
+    - ruby <2.6, >=2.4.0
+  run:
+    - ruby <2.6, >=2.4.0
+
+test:
+  commands:
+    - ruby -r {{ name }} -e 'exit 0'
+
+about:
+  home: https://rubygems.org/gems/{{ name }}
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE.txt
+  summary: A modular Ruby web server interface
+  doc_url: https://www.rubydoc.info/gems/{{ name }}
+  dev_url: https://github.com/rack/{{ name }}
+
+extra:
+  recipe-maintainers:
+    - svengiegerich
```

happy to co-maintain. :)

```suggestion
    - svengiegerich
    - janjagusch
```